sdk/identity/identity/src/credentials/authorizationCodeCredential.browser.ts

```diff
@@ -34,13 +36,15 @@ export class AuthorizationCodeCredential implements TokenCredential {
     redirectUriOrOptions: string | TokenCredentialOptions | undefined,
     options?: TokenCredentialOptions
   ) {
+    logger.error(BrowserNotSupportedError);
     throw BrowserNotSupportedError;
   }
 
   public getToken(
     scopes: string | string[],
     options?: GetTokenOptions
   ): Promise<AccessToken | null> {
+    logger.getToken.error(BrowserNotSupportedError);
```

Comment:
I might be missing it, but does this getToken call not need a success log message?

Reply:
Some of our credentials don't currently support the browsers! This is a special case for JavaScript.

Reply:
Got it. Thanks for the info :)